genie-docs/src/docs/asciidoc/_metrics.adoc

```diff
@@ -197,6 +197,30 @@ It will have to be added if one is desired otherwise metrics are just published
 |ArchiveStatusCleanupTask
 |status, exceptionClass
 
+|genie.jobs.attachments.s3.count.distribution
+|Distribution summary of for the number of files attached
+|count
+|S3AttachmentService
```

Comment:
By convention we've generally added `Impl` to the end of these implementations of various interfaces or at least we've tried to I think